openstudiocore/src/utilities/bcl/LocalBCL.cpp

```diff
@@ -368,12 +380,12 @@ namespace openstudio{
   std::vector<BCLMeasure> LocalBCL::measures() const
   {
     std::vector<BCLMeasure> allMeasures;
-    QSqlDatabase database = QSqlDatabase::database(m_libraryPath+m_dbName);
+    auto database = getDatabase();
```

Comment:
@lefticus will this make it harder to find/replace instances of QSqlDatabase?  I guess the intent is to write code that can remain after the refactor?

Reply:
I found myself changing the exact same line of code 20 times and figured it would make it easier for the next phase of refactor if you only had 1 line to change here.